winbuild/build_prepare.py

```diff
@@ -257,24 +257,25 @@ def cmd_msbuild(
         "libs": [r"bin\*.lib"],
     },
     "libimagequant": {
-        # Merge master into msvc (matches 2.14.1 except for version bump)
-        "url": "https://github.com/ImageOptim/libimagequant/archive/16adaded22d1f90db5c9154a06d00a8b672ca09a.zip",  # noqa: E501
-        "filename": "libimagequant-16adaded22d1f90db5c9154a06d00a8b672ca09a.zip",
-        "dir": "libimagequant-16adaded22d1f90db5c9154a06d00a8b672ca09a",
+        # commit: Merge branch 'master' into msvc (matches 2.16.0 tag)
```

Comment:
If we're using a tag, can we download the actual tag archive instead of a commit hash?

From https://github.com/ImageOptim/libimagequant/releases/tag/2.16.0:

https://github.com/ImageOptim/libimagequant/archive/refs/tags/2.16.0.zip

Reply:
This is a commit merging the 2.16.0 tag into the msvc branch, there are a few minor changes: https://github.com/ImageOptim/libimagequant/compare/2.16.0...f41ee301ff3a407b16991af3dbe03910919bbdc3